tests/integration_tests_control_files/advection_limiter/1D_advection_limiter.prm

```diff
@@ -0,0 +1,49 @@
+# -------------------
+
+set test_type = advection_limiter
+
+# Number of dimensions
+set dimension = 1
+
+set use_weak_form = false
+set flux_nodes_type = GLL
+
+# Strong DG - LaxF
+#set use_split_form = false
+#set conv_num_flux = lax_friedrichs
+
+# NSFR - Ra
+set use_split_form = true
+set two_point_num_flux_type = Ra
+set conv_num_flux = two_point_flux_with_lax_friedrichs_dissipation
+
+# The PDE we want to solve
+set pde_type = advection
+
+subsection limiter
+  set bound_preserving_limiter = maximum_principle
+end
+
+set do_renumber_dofs = false
+
+subsection ODE solver
+
+  set ode_output = verbose
+  
+  set nonlinear_max_iterations = 500000
+
+  set print_iteration_modulo = 100
+
+  set ode_solver_type = runge_kutta
+
+  set initial_time_step = 0.0001
+
+  set runge_kutta_method = ssprk3_ex
+
+  set output_solution_every_x_steps = 1
```

Comment:
Does outputting the solution this often create a lot of files or take up significant storage space? It may be better to output fewer files in the version committed to master, as it will be run as a part of `ctest` for everyone. I typically also commit "pruned" versions of my tests when they are long, as we want them to fail in the case that an unintended change has been made and don't necessarily need full results. But this is at your discretion, as all of your tests are 1D so probably quite short!

Reply:
I see that you've left some tests commented in the `CMakeLists.txt`, that's a good way to make sure that `ctest` doesn't get bloated!

Reply:
Modified all prm files to set output to every 100 steps.